schemas/sdk_metadata.sql

```diff
@@ -64,8 +64,29 @@ INSERT INTO sdk_type_info (type, description) VALUES
-    ('u2c', 'The concept of Users is replaced with Contexts, which can be used to represent users, devices, or other entities.'),
-    ('hooks', 'Hooks are collections of user-defined callbacks that are executed by the SDK at various points of interest, usually for metrics or tracing.');
+    ('appConfig', 'Application metadata', 'Specify application and application version information.'),
+    ('autoEnvAttrs', 'Automatic environment attributes', 'Automatically include device and application data in each evaluated context.'),
+    ('bigSegments', 'Big segments', 'Configure a persistent store to hold segments that are either synced from external tools, or that contain an arbitrarily large number of contexts of any one context kind.'),
+    ('bootstapping', 'Bootstrapping', 'Provide an initial set of flag values that are immediately available during client initialization.'),
+    ('contexts', 'Contexts and context kinds', 'Evaluate flags based on contexts. A context is a generalized way of referring to the people, services, machines, or other resources that encounter feature flags. SDKs without this feature only support flag evaluation for users.'),
+    ('experimentation', 'Experimentation', 'Connect a flag with one or more metrics to measure end-user behavior for different variations of a flag. Requires minimum SDK versions, but no SDK configuration.'),
+    ('variationDetail', 'Flag evaluation reasons', 'Receive information about how a flag variation was calculated, for example, because it matched a specific targeting rule.'),
+    ('allFlags', 'Getting all flags', 'Return the flag variations for all feature flags for a given context.'),
+    ('hooks', 'Hooks', 'Define callbacks that are executed by the SDK at various points of interest, usually for metrics or tracing.'),
+    ('migrations', 'Migration flags', 'Configure how to read and write data for an old and new system, determine which stage of a migration the application is in, execute the correct read and write calls for each stage.'),
+    ('multiEnv', 'Multiple environments', 'Evaluate flags from multiple environments using a single client instance'),
+    ('offlineMode', 'Offline mode', 'Close the SDK''s connection to LaunchDarkly. Use cached or fallback values for each flag evaluation.'),
+    ('otel', 'OpenTelemetry', 'Add flag evaluation information to OpenTelemetry spans.'),
+    ('privateAttrs', 'Private attributes', 'Use context attribute values for targeting, but do not send them to LaunchDarkly.'),
+    ('flagsFromFile', 'Reading flags from a file', 'Use flag values, specified in JSON or YAML files, for all flag evaluations. Useful for testing or prototyping; do not use in production.'),
+    ('relayProxyDaemon', 'Relay Proxy in daemon mode', 'Configure the SDK to connect to the Relay Proxy''s data store.'),
+    ('relayProxyProxy', 'Relay Proxy in proxy mode', 'Configure the SDK to connect to a load balancer set up in front of several Relay Proxy instances.'),
```

Comment:
While this is a good idea (load balancer), you can also directly connect to Relay in proxy mode. Doesn't really matter though since we can update this text whenever we want.

Reply:
What do you recommend instead? I feel like "Configure the SDK to connect to the Relay Proxy in proxy mode" isn't very helpful

Reply:
Some thoughts I had about these that I'm remembering now. There is a bit of nuance here.

It's almost like `relayProxyProxy` is not a real feature. Once an SDK supports streaming/polling we don't need to add anything special to make Relay in proxy mode work. 

So perhaps instead we have `streamingDataSource` and `pollingDataSource` features and derive the relay proxy related "features" from the presence or absence of those?

For `relayProxyDaemon`, it's really more a generic capability similar to `fileDataSource` or `testDataSource`: "redisDataSource" or "dynamoDataSource" or more generically "persistentStore". The fact that Relay can be populating that store is usually relevant, but it's not required. 

I get that a customer really wants to know "Does this work with Relay" and we want to put a checkbox there. So we should still have it, but maybe it can be derived instead of specified.

Reply:
For now, updated the descriptions to be slightly more generic. Happy to replace these entries with something derived instead of specified in the future, but would like that to be followup work (and would like to volunteer someone else! lol -- I'm not sure how to approach this).

Reply:
Ok sounds good, I'll take a crack at "refactoring" them later.